zfit/minimizers/errors.py

```diff
@@ -0,0 +1,199 @@
+import numpy as np
+from scipy import optimize
+from ..param import set_values
+from ..util.container import convert_to_container
+
+
+class NewMinimum(Exception):
+    """Exception class for cases where a new minimum is found."""
+    pass
+
+
+def pll(minimizer, loss, params, values) -> float:
+    """Compute minimum profile likelihood for given parameters and values."""
+    params = convert_to_container(params)
+    values = convert_to_container(values)
+
+    verbosity = minimizer.verbosity
+    minimizer.verbosity = 0
+
+    with set_values(params, values):
+        for param in params:
+            param.floating = False
+        minimum = minimizer.minimize(loss=loss)
+
+    for param in params:
+        param.floating = True
+    minimizer.verbosity = verbosity
+
+    return minimum.fmin
+
+
+def set_params_to_result(params, result):
+    """Set parameters values to the values in the fitted value in the result."""
+    for param in params:
+        param.set_value(result.params[param]["value"])
+
+
+def get_crossing_value(result, params, direction, sigma, rootf, rtol):
+    """Find the crossing point between the profiled loss function, for given parameters, and the value of
+    `errordef` for a given direction (positive / negative).
+    `errordef` = 1 for a chisquare fit, = 0.5 for a likelihood fit.
+    """
+
+    all_params = list(result.params.keys())
+    loss = result.loss
+    errordef = loss.errordef
+    fmin = result.fmin
+    minimizer = result.minimizer.copy()
+    minimizer.tolerance = minimizer.tolerance * 0.5
+    rtol *= errordef
+
+    set_params_to_result(all_params, result)
```

Comment:
```suggestion
    set_values(all_params, result)
```
We had the same idea, I've added this functionality independently to `set_values` :)

Reply:
I yes I saw the new feature,  it will be useful in `hepstats` :D